book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Getter and Setter for `@rentals` as for `@title` and `@author`, `rentals` should be `attr_accessor` as well. Kindly fix that 🙏🏾

Reply:
Done too ✨